sql/database.sql

```diff
@@ -1638,7 +1638,7 @@ CREATE TABLE  `facility_user_ids` (
   `field_value` TEXT,
   PRIMARY KEY (`id`),
   KEY `uid` (`uid`,`facility_id`,`field_id`),
-  KEY `uuid` (`uuid`)
+  UNIQUE KEY `uuid` (`uuid`)
```

Comment:
This should not be UNIQUE since it is a vertical table
(hint: check out the data with several entries in it and you'll see what I mean :) )

Reply:
Oh ya, I remember those duplicate entries with the same uuid. Will revert this :+1: